cmd/dep/testdata/harness_tests/init/case1/final/lock.toml

```diff
@@ -0,0 +1,12 @@
+memo = ""
```

Comment:
I am not sure why the memo is empty? 😄  It's been like that for a while; here's there current `lock.json` for that test:

https://github.com/golang/dep/blob/master/cmd/dep/testdata/harness_tests/init/case1/final/lock.json#L2

Think that's a bug?